admin/class-boldgrid-backup-admin-archive-details.php

```diff
@@ -131,8 +131,13 @@ public function render_archive() {
 			$this->core->delete_archive_file();
 		}
 
+		if ( ! empty( $_FILES['file'] ) ) {
```

Comment:
This code is repeated many times. I think we should try to be dry - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

If someone creates a new admin page they could easily forget that this code needs to be added to every admin page, and easily introduce a bug.

What if we use logic like this, and have it live only one place in the plugin?

```php
if ( is_total_upkeep_page() and is_uploading_backup_file() ) {
    $this->core->upload->upload_archive_file();
}
```